src/main/resources/application.yaml

```diff
@@ -1096,6 +1103,7 @@ nbcEmailAddress: ${IA_NBC_EMAIL}
 ctscEmailAddress: ${IA_CTSC_EMAIL}
 ctscAdminPendingPaymentEmailAddress: ${IA_CTSC_ADMIN_PENDING_PAYMENT_EMAIL:pendingPayment@example.com}
 bailHomeOfficeEmailAddress: ${IA_BAIL_HOME_OFFICE_EMAIL:ho-bail-alar@example.com}
+bailHomeOfficeEmailAddressForApplicationSubmittedEdited: ${IA_BAIL_HOME_OFFICE_EMAIL_APP_EDITED_SUBMITTED:ALARAppealsBailTeam@homeoffice.gov.uk}
```

Comment:
Does this not mean if we test in DEV/AAT/DEMO we're going to send to the real email? we should only have dummy emails here. 
Then add the real email in secret vaults in different ENV via bootstrap, Jenkinsfile_CNP, Jenkinsfile_nightly, Jenkinsfile_para

Reply:
${IA_BAIL_HOME_OFFICE_EMAIL:[ALARAppealsBailTeam@homeoffice.gov.uk]

Reply:
I've deleted that line and am using `bailHomeOfficeEmailAddress` like in all other HO personalizations